Core/Utilities/TimeLeft/LSFTimeLeft.py

```diff
@@ -146,10 +146,12 @@ def __init__( self ):
         else:
           self.log.error( 'Cannot source the LSF environment', ret['Message'] )
     if not self.normRef:
+      # If nothing works take this as the unit
+      self.normRef = 1.
       # If nothing worked, take the maximum defined for a Model
-      if modelMaxNorm:
-        self.normRef = modelMaxNorm
-        self.log.info( 'Reference Normalization taken from Max Model:', self.normRef )
+      # if modelMaxNorm:
+      #  self.normRef = modelMaxNorm
+      #  self.log.info( 'Reference Normalization taken from Max Model:', self.normRef )
```

Comment:
You are completely right: it makes no sense anyway to assume the normalisation is that of the fastest machine ... I just keep the calculation in case ;-)